rollout/trafficrouting/istio/istio.go

```diff
@@ -1499,7 +1499,9 @@ func (r *Reconciler) RemoveManagedRoutes() error {
 			return fmt.Errorf("[RemoveManagedRoutes] failed to get http routes from virtual service: %w", err)
 		}
 		if !found {
-			return fmt.Errorf("[RemoveManagedRoutes] %s: %w", SpecHttpNotFound, err)
+			// Nothing to do here so return early this can be hit when only tls routes are defined
```

Comment:
dont we need to check for tls enabled or not?

Reply:
Should not need to, RemoveManagedRoutes only works with http routes because istio does not support any type of header, mirroring etc with tls only routes because it is encrypted. So the code was changed to not return an error if there is only tls routes defined but instead just return nil allowing the rest of the reconciliation to continue.

Reply:
oh..k. got it..thanks for the explanation